U8HF15/AI_Practice/Commands.cs

```diff
@@ -0,0 +1,80 @@
+﻿using AI_Practice.CommandFiles;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Dataflow;
+using System.Windows.Input;
+
+namespace AI_Practice
+{
+
+     static class Commands
+    {
+
+
+        public static Dictionary<string, CommandInterface> COMMANDS = new();
+        public static Network AI;
+        static Commands() {
+
+
+            var commands = AppDomain.CurrentDomain.GetAssemblies().SelectMany(x => x.GetTypes()).Where(y => typeof(CommandInterface).IsAssignableFrom(y)&& !y.IsInterface && !y.IsAbstract);
```

Comment:
Reflecton 👍, de a Az AppDomain.CurrentDomain-ből lekérni minden Assembly-t necces. Lehet akár security issue is. 👎